Xamarin.Forms.Core/Routing.cs

```diff
@@ -7,25 +7,30 @@ namespace Xamarin.Forms
 	public static class Routing
 	{
 		static int s_routeCount = 0;
+
+#if NETSTANDARD1_0
 		static Dictionary<string, RouteFactory> s_routes = new Dictionary<string, RouteFactory>();
+#else
+		static Dictionary<string, RouteFactory> s_routes = new Dictionary<string, RouteFactory>(StringComparer.InvariantCultureIgnoreCase);
```

Comment:
So for this one we're actually making it all case sensitive so any to lowers or ignore cases can be removed.